traveler-swift-core/TravelerKit/AuthPath.swift

```diff
@@ -21,6 +21,8 @@ enum AuthPath {
     case cancellationQuote(Order)
     case cancelOrder(CancellationQuote)
     case emailOrderConfirmation(Order)
+    case wishlistToggle([CatalogItem], travelerId: String)
```

Comment:
Should this take in `[Product]` instead? I as a dev want to be able to do this both at the catalog level and the details level